tests/integration/test_source_git.py

```diff
@@ -75,6 +76,14 @@ def test_basic_local_update_empty_patch(
     assert not spec.patches["not_applied"]
 
 
+def diff_strings(a: str, b: str):
```

Comment:
Is this used anywhere?

Reply:
I used it to diff strings below - pytest -vv wasn't helpful there at all, and wanted to keep it for future; dropped it since it's trivial to get back